### ISSUE TITLE
Project Name Conflicts

### ISSUE THREAD

**Issue by friktor** [yoshuawuyts/tide#1]
At the moment I already have an existing Rust project called "Rise" -- the GUI library. For the project there is a registered domain rise.rs, and an organization "rise-ui". I would like to avoid any confusion in the future, so let's discuss how would we solve the problem with name conflicts. I did not reserve the project name on crates.io because rise-ui has github webrender dependency, and crates.io does not allow the publication of a project with git dependencies. Though some independent libs like [raqm](https://crates.io/crates/raqm) have already been published on crates.io on behalf of the project, with keywords like "rise, rise-ui".

##### Proofs
* [organization on github](https://github.com/rise-ui)
* [commit when the project was named a rise](https://github.com/rise-ui/rise/blob/878ca562f5c1188b41d86783dbe9771b5ba2af28/Cargo.toml)
* [domain ownership confirmation](https://image.ibb.co/jXB3wU/Screenshot_at_Aug_10_09_20_55.png)

**Comment by aturon** [comment]
Hi @friktor, thanks for the heads up about this! I had searched for web frameworks (in any language) and crates named Rise, but looks like that wasn't enough :joy:

I'll work with @yoshuawuyts and the rest of the WG to figure out an alternative and we can transfer ownership of the name on crates.io

**Comment by yoshuawuyts** [comment]
@friktor we've renamed the project to `tide`, as in "the rising _tide_" :tada: -- I've also invited you to be an owner of `tide` on crates.io. The only inconvenience is that the `0.1.0` version is already taken, but hopefully that's not too much of a problem.

All in all I hope this ends up being an adequate resolution. Thanks for raising this; excited to see what comes out of both `rise` and `tide` :sparkles:
